File: TwoSumII.cs
public class Solution {
    public int[] TwoSumII(int[] numbers, int target) {
        
        int i = 0, j = numbers.Length - 1;
        
        while(i < j)
        {
            int sum = numbers[i] + numbers[j];
            
            if (sum == target) {
                return new int[]{i+1,j+1};
            }
            else if (sum < target) {
                i++;
            }
            else{
                j--;
            }
        }
        
        return new int[0];
    }
}

File: Python/twoSumII.py
class Solution:
    def twoSumII(self, numbers: List[int], target: int) -> List[int]:

        left = 0
        right = len(numbers) - 1

        while left < right:
            test = numbers[left] + numbers[right]

            if test > target:
                right -= 1
            
            elif test < target:
                left += 1

            else:
                return [left + 1, right + 1]